medications/README.md

```diff
@@ -0,0 +1,152 @@
+| component / ehrSupplyPrescribe / availabilityTime \[@value]                              | Mapped from `MedicationRequest.dispenseRequest.validityPeriod.start`                                                                                                                                                                                                                                                                                                    |
+| component / ehrSupplyPrescribe / quantity \[@value]                                      | If present then: `MedicationRequest.dispenseRequest.quantity.value` otherwise defaults to `"1"`                                                                                                                                                                                                                                                                         |
+| component / ehrSupplyPrescribe / quantity \[@unit]                                       | If present then: `MedicationRequest.dispenseRequest.quantity.value`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.extension`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.quantity.extension`.<br/>If none present then default value is `"Unk UoM"` (unknown unit of measurement).                                          |
+| component / ehrSupplyPrescribe / quantity / translation \[@value]                        | If present then: `MedicationRequest.dispenseRequest.quantity.value` otherwise defaults to `"1"`                                                                                                                                                                                                                                                                         |
+| component / ehrSupplyPrescribe / quantity / translation / originalText                   | If present then: `MedicationRequest.dispenseRequest.quantity.value`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.extension`.<br/>Otherwise if present then: `MedicationRequest.dispenseRequest.quantity.extension`.<br/>If none present then default value is `"Unk UoM"` (unknown unit of measurement).                                          |
+| component / ehrSupplyPrescribe / inFulfillmentOf \[@typeCode]                            | fixed value = `"FLFS"`                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Same as previous comment, fixed in the MIM, so it might be easier to read if we rove the ones that are?